pkg/plugins/resource/resource.go

```diff
@@ -0,0 +1,171 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+	"open-cluster-management.io/placement/pkg/plugins"
+)
+
+const (
+	placementLabel = "cluster.open-cluster-management.io/placement"
```

Comment:
nit: you can use clusterv1alaph1.PlacementLabel now.

Reply:
Updated.